src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js

```diff
@@ -867,13 +893,43 @@ describe('test suite for synchronous action creators', () => {
     expect(store.getActions()).toEqual(expectedActions);
   });
 
+  it('should handle ADD_NEW_SET_ROW', () => {
```

Comment:
What is the motive behind adding "handle" to all these and below?

Reply:
Improved the arrangement and naming of these tests.